### DISCUSSION TITLE
Format problem

### DISCUSSION THREAD

**Discussion by The-TallGuy** [Tyrrrz/DiscordChatExporter#1195] [Q&A]
So I made some exports a few months ago and I need to use them now. I still have the files, but they are HTML, not JSON...Can I convert it in any way to use the DiscordChatImporter tool, or am I just out of luck?

**Comment by Yudi** [comment]
DiscordChatExporter can't convert files

Additionally, HTML doesn't have the same data JSON has, so some info would be missing if you converted it

(#1154, #1047)

**Reply by The-TallGuy** [reply]
Ok. I suspected there would be some differences but thanks for confirming.
Also, I know the exporter can't do the conversion, just wondering if there's a way to do it so I could do the importing. I really don't care about the data lost (unless the json would have also saved the images, cuz that would be double sad), I just want to put the small data I have from the channels back.

**Reply by Yudi** [reply]
[Stoxis made a Python script](https://github.com/Tyrrrz/DiscordChatExporter/discussions/1047#discussioncomment-6870050) but I haven't tested it

**Reply by The-TallGuy** [reply]
Thank you Yudi, you're a lifesaver 🙏🏻

**Reply by Yudi** [reply]
Thank @Stoxis :-)

**Reply by The-TallGuy** [reply]
Thank you @Stoxis
